### DISCUSSION TITLE
My Github Copilot was suddenly unavailable. I have no idea what happened.  Can someone help me to ascertain?

### DISCUSSION THREAD

**Discussion by shiyishiaa** [community/community#123604] [GitHub Education]
### Hi everyone,

My Github Copilot was suddenly unavailable about a week ago, for which I have no idea what happened. :(
I appreciate you in advance if there is any help.
Besides, if there is any mistake I have made, please let me know.

**Comment by PingWang28** [comment]
I also meet this issue, but finally I find github need me reverify my edu identify . You can check your mail first.  But after I following the step which mentioned in the mail , I can't find a place where to resubmit my informaton.  Let's see what you will meet .

**Reply by shiyishiaa** [reply]
Thanks for your help. <3 However, I have not gotten anything from GitHub.

**Comment by shiyishiaa** [comment]
Sadly, still cannot retrieve my entitlement to access the Github Copilot. :-(
